src/views/Login.jsx

```diff
@@ -77,11 +77,11 @@ class Login extends Component {
         />
         <br/>
         <RaisedButton
-          onSubmit={this.handleOnSubmit}
+          onClick={this.onSubmit}
```

Comment:
this should be `onTouchTap`. http://www.material-ui.com/#/components/raised-button